src/plugin.ts

```diff
@@ -1,16 +1,9 @@
 import { SanitizedConfig } from 'payload/config'
 import { UploadedFile } from 'express-fileupload'
-import { SanitizedCollectionConfig, CollectionBeforeChangeHook, CollectionAfterDeleteHook } from 'payload/types'
+import { CollectionBeforeChangeHook, CollectionAfterDeleteHook } from 'payload/types'
+// @ts-ignore
 import { APIError } from 'payload/errors'
-import { AdapterInterface } from './payload-plugin-s3'
-
-function isUploadedFile (object: unknown): object is UploadedFile {
-  if (object !== null && typeof object === 'object') {
-    return 'mimetype' in object
-  }
-
-  return false
-}
+import { AdapterInterface, S3PluginCollectionModifiers } from './payload-plugin-s3'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/richardvanbergen/payload-plugin-cloud-storage/pullRequest?prid=7829468)
```suggestion
import { AdapterInterface, S3PluginCollectionModifiers } from "./payload-plugin-s3"
```